lib/eslint.js

```diff
@@ -144,6 +145,7 @@ function addDeclaredGlobals(program, globalScope, config) {
         }
     });
 
+    assign(exportedGlobals, config.exported);
```

Comment:
Yeah, so I'm not even sure it matters that we copy here. I could probably just remove this, because we're just reading the list and marking some thing in the list as used. It was mostly used just because that's how the rest of the code in that section was written. Up to you how to proceed